Jenkinsfile_nightly

```diff
@@ -50,8 +50,7 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
 
 withNightlyPipeline("nodejs", product, component) {
     env.TEST_URL = params.URL_TO_TEST
-    env.TEST_E2E_URL = params.URL_TO_TEST
-    env.PATH_TO_TEST_FILES =  params.test_path
+    env.FE_E2E_TEST_PATH_TO_RUN =  params.test_path
```

Comment:
doublecheck with Phani that this isn't going to overlap his changes

Reply:
checked and it's not a problem as Phani is focussed on a different repo (probate-caveat-fronend)